README.rst

```diff
@@ -0,0 +1,52 @@
+
+CellML Specification
+====================
+
+This repository will be used by the CellML editors to develop the CellML specification (starting with version 1.2)
+The content has been reworked to include an informal guide embedded within the normative specification text (in version 2.0).
+
+If things work correctly, changes in this repository will be reflected over at: https://cellml-specification.readthedocs.org/.
+
+Building the documentation locally
+----------------------------------
+
+The documentation can be built on your local machine with the following instructions.
+Do take note that the following instructions are suitable for Unix-derived operating systems and may need modification for other operating systems.
+Also, note that we require at least version 3.5 of Python to build the documentation.
+
+First clone a local copy of the CellML specification repository::
+
+  git clone https://github.com/cellml/cellml-specification.git
+
+Next setup a virtual environment::
+
+  virtualenv venv_specification
+
+Note: Make sure you are creating a Python 3 virtual environment, if required use the -p flag to set the Python interpreter (for example `virtualenv -p /absolute/path/to/python venv_specification`).
+
+Then activate the virtual environment and instal the required packages::
+
+  source venv_specification/bin/activate
+  pip install sphinx sphinx_rtd_theme
+  pip install -r cellml-specification/requirements.txt
+
+That completes the environment setup required for building the documentation.
+Once the environment is completed the html form of the documentation can be built with the following commands::
```

Comment:
Why is it back to `html` (and `pdf` later on)?!

Reply:
@kerimoyle commented that it should be such. I have realised that one person has said one thing and another the opposite so which is it CAPS or not?

Reply:
Or maybe I just can't read English

Reply:
No, I think she realised that I had already met a similar comment about converting `html` to `HTML` and `pdf` to `PDF`, so she just wanted you (I think) to concentrate on the "whilst" bit... unless I am the one who can't read English, which would be better for you... :)

Reply:
Yep, sorry for the confusion.  CAPS it is ...